lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -617,9 +618,16 @@ def get_distribution_DragonFly(self):
 
     def get_distribution_NetBSD(self):
         netbsd_facts = {}
-        # FIXME: poking at self.facts, should eventually make these each a collector
         platform_release = platform.release()
-        netbsd_facts['distribution_major_version'] = platform_release.split('.')[0]
+        netbsd_facts['distribution_release'] = platform_release
```

Comment:
I know that `platform.release()` is used for other BSDs, but I'm wondering if that is actually the expected value. I'm not a BSD user, but at least in the Linux world, I expected "release" to be a word, such as "trusty", or "stretch" rather than a version number.

Maybe it should be `GENERIC` as returned from `sysctl -n kern.version` rather than using the output of `platform.release()`?